src/main/resources/templates/section.html

```diff
@@ -8,13 +8,13 @@
 </head>
 <body>
 <div layout:fragment="content">
-    <h1>Section <span th:text="${section.number}"></span></h1>
+    <h1>Section <span th:text="${section.number + ' ' + section.course.getName()}"></span></h1>
     <div class="container">
         <h6>Room:</h6>
         <div class="container">
             <table class="table">
                 <tr >
-                    <td th:text="${section.room?.name + ' ' + section.room?.building?.name}"></td>
+                    <td th:text="${section.room?.name + ' ' + section.room?.getBuilding().getName()}"></td>
```

Comment:
I think this change is vulnerable to getting a null value, and thus vulnerable to crashing the html. BUT if you assure me that "it actually is safe" then I will gladly accept it

Reply:
ok I will add the `?`